docs/api/alerting/list_rule_types.asciidoc

```diff
@@ -4,7 +4,7 @@
 <titleabbrev>List rule types</titleabbrev>
 ++++
 
-Retrieve a list of alerting rule types that the user is authorized to access.
+Retrieve a list of Alerting rule types that the user is authorized to access.
```

Comment:
Should we be consistent here by adding `{kib} Alerting`?

Reply:
Updated in https://github.com/elastic/kibana/pull/119314/commits/f8dcbce022353d226332a67391e98c9d37159051

Reply:
Gail liked it better without {kib} so it's back to just `Alerting`.

Reply:
nit: Maybe we can skip word alerting here? Just a thought.